mutiny-core/src/nodemanager.rs

```diff
@@ -455,39 +455,41 @@ impl NodeManager {
         Ok(nm)
     }
 
-    pub(crate) fn start_redshifts(nm: Arc<NodeManager>) {
-        let node_manager = nm.clone();
```

Comment:
very confused on how i was able to remove this, will do a test redshift to make sure it still works